src/components/application_manager/test/resumption/resumption_data_test.cc

```diff
@@ -52,6 +52,52 @@ using ::testing::Return;
 using ::testing::ReturnPointee;
 using ::testing::ReturnRef;
 
+ResumptionDataTest::~ResumptionDataTest() {
+  for (auto& submenu : test_submenu_map) {
+    delete submenu.second;
```

Comment:
@SNiukalov why do we dlete only second item? Don't we clear the entire map?

Reply:
@AByzhynar 
Fixed in:https://github.com/LuxoftSDL/sdl_core/pull/41/commits/be4a738c2392d04c4fd0cd9a2dee39ce76b94c6e